src/app/shared/components/personal-info-modal/personal-info-modal.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="logo">
+  <img src="assets/img/logo.svg" alt="Логотип" />
+</div>
+
+<div class="message-container">
+  <p class="message-text" *ngIf="source === openedFrom.OrderPage">
+    Для отправки заявки укажите Ваши персональные данные
+  </p>
+</div>
+
+<mat-tab-group mat-align-tabs="center">
+  <mat-tab label="УКАЖИТЕ ВАШИ ПЕРСОНАЛЬНЫЕ ДАННЫЕ">
+    <form #personalInfoForm="ngForm">
+      <mat-form-field floatLabel="auto">
+        <mat-label>Имя</mat-label>
+        <input
+          type="text"
+          name="name"
+          matInput
+          minlength="2"
+          maxlength="49"
+          required
+          [(ngModel)]="contacts.name"
+          lcValidationErrors
+        />
+      </mat-form-field>
+
+      <mat-form-field floatLabel="auto">
+        <mat-label>Фамилия</mat-label>
+        <input
+          type="text"
+          name="surname"
+          matInput
+          minlength="2"
+          maxlength="49"
+          required
+          [(ngModel)]="contacts.surname"
+          lcValidationErrors
+        />
+      </mat-form-field>
+
+      <mat-form-field floatLabel="auto">
+        <mat-label>Телефон</mat-label>
+        <input
+          type="tel"
+          name="phone"
+          matInput
+          minlength="11"
+          maxlength="11"
+          required
+          [(ngModel)]="contacts.phone"
+          lcPhoneValidator
+          lcValidationErrors
```

Comment:
the fields displayed with validation errors

Reply:
done